### PR TITLE
refactor(components): [tour] replace rect with path

### PR DESCRIPTION
**Pull request by btea**
Please make sure these boxes are checked before submitting your PR, thank you!

- [ ] Make sure you follow contributing guide [English](https://github.com/element-plus/element-plus/blob/master/.github/CONTRIBUTING.en-US.md) | ([中文](https://github.com/element-plus/element-plus/blob/master/.github/CONTRIBUTING.zh-CN.md) | [Español](https://github.com/element-plus/element-plus/blob/master/.github/CONTRIBUTING.es.md) | [Français](https://github.com/element-plus/element-plus/blob/master/.github/CONTRIBUTING.fr-FR.md)).
- [ ] Make sure you are merging your commits to `dev` branch.
- [ ] Add some descriptions and refer to relative issues for your PR.

### CODE REVIEW

**Comment by btea** [comment]
@Fuphoenixes Hello, could you help review this PR?

File: packages/components/tour/src/mask.vue [hunk 1]
```diff
@@ -3,14 +3,14 @@
...
-      pointerEvents: pos && targetAreaClickable ? 'none' : 'auto',
-    } as any)"
+    position: 'fixed',
+    left: 0,
+    right: 0,
+    top: 0,
+    bottom: 0,
+    zIndex,
+    pointerEvents: pos && targetAreaClickable ? 'none' : 'auto',
+  } as any)"
```
Comment (ryuhangyeong) [inline packages/components/tour/src/mask.vue:13]: There seems to be a problem with the code formatting.

**Review by ryuhangyeong — APPROVED** [review]

File: packages/components/tour/src/mask.vue (hunk 1 above)
Comment (btea) [reply packages/components/tour/src/mask.vue:13]: This is what happens after I execute `pnpm format`.

Comment (ryuhangyeong) [reply packages/components/tour/src/mask.vue:13]: OK

**Comment by ryuhangyeong** [comment]
@btea Thank you for hard work

**Comment by btea** [comment]
@ryuhangyeong ❤️

**Comment by ryuhangyeong** [comment]
@btea  

I have a question.

@jw-foss

Does he no longer maintain element-plus? 😭

Or is it just a short break?

I'm curious about our roadmap and goals for 2024. I don't know who to ask.

**Comment by kooriookami** [comment]
@ryuhangyeong We use WeChat to communicate, it's a Chinese chatting software, and overseas version available. Do you want to join our group.

**Comment by ryuhangyeong** [comment]
of course!

I think it's a good idea to join the group as I will continue to contribute to the ep.

Can you send the invitation link to the email below?

webfrontend@naver.com

**Comment by kooriookami** [comment]
@ryuhangyeong You need to download wechat from your app store or https://www.wechat.com. And you need to register an account. I will send you an invitation qrcode later.

**Comment by ryuhangyeong** [comment]
@kooriookami 

Yes, it may take some time as I am not familiar with that software. I will mention you again when preparations are complete.

thank you.

**Comment by kooriookami** [comment]
OK, I sent you the email already, any questions you can ask me.

**Comment by ryuhangyeong** [comment]
@kooriookami Blocked from WeChat. Did I do something wrong?

**Comment by kooriookami** [comment]
> @kooriookami Blocked from WeChat. Did I do something wrong?

What? Did wechat tell you the reason?

**Comment by ryuhangyeong** [comment]
@kooriookami @btea 

reason

"This account has been blocked for violations such as harassment, harmful marketing, or fraud."

**Comment by kooriookami** [comment]
@ryuhangyeong Did you file a complaint?

**Comment by ryuhangyeong** [comment]
@kooriookami 

no never

**Comment by ryuhangyeong** [comment]
@kooriookami no never

**Comment by kooriookami** [comment]
You can try to submit a ticket or ask customer service. I can provide evidence because I pull you in the group.

**Comment by ryuhangyeong** [comment]
> You can try to submit a ticket or ask customer service. I can provide evidence because I pull you in the group.



Can you help me?

**Comment by kooriookami** [comment]
@ryuhangyeong Sure, I'll send you my phone number and wechat number.

**Pull request merged**